direct_chat.py

```
@@ -1,892 +1 @@
-        raise HTTPException(status_code=404, detail="Session not found")
-
-    # Store resume payload in session; the background job is polling for this
-    _direct_chat_store.update_resume_payload(session_id, req.model_dump())
-
-    # Also log the user's response in history for continuity
-    msg = f"Action: {req.action}"
-    if req.input: msg += f" - Input: {req.input}"
-    _direct_chat_store.append_message(session_id, "user", msg)
-
-    return {"status": "resumed", "session_id": session_id}
+{}
```

Ruff (0.15.18)

[warning] The file is executable but no shebang is present

(EXE002)